packages/ui/src/components/Form/CustomAutoFields.tsx

```diff
@@ -35,7 +35,7 @@ export function CustomAutoFields({
   const actualFields = (fields ?? schema.getSubfields()).filter(
     (field) =>
       !omitFields!.includes(field) &&
-      (field === 'parameters' || field.toLowerCase().includes(filteredFieldText.toLowerCase())),
+      (field === 'parameters' || field.toLowerCase().includes(filteredFieldText.replace(/\s/g, '').toLowerCase())),

```

Comment:
@tplevko, in this case, we're recreating the regexp on every cycle which is expensive, would be possible to create the regexp once in a shared service and then use it from there?

That being said, since we're applying it to the `filteredFieldText` string, perhaps we can prepare the string once before the `includes`, perhaps something like:

```typescript
  const cleanQueryTerm = filteredFieldText.replace(/\s/g, '').toLowerCase();
  const actualFields = (fields ?? schema.getSubfields()).filter(
    (field) =>
      !omitFields!.includes(field) &&
      (field === 'parameters' || field.toLowerCase().includes(cleanQueryTerm)),
  );
```

Reply:
Thanks for the hint. Updated according your recommendation.